.github/workflows/release.yml

```
@@ -98,6 +98,32 @@ jobs:
+        uses: actions/checkout@3d3c42e5aac5ba805825da76410c181273ba90b1 # v7.0.1
+        with:
+          ref: ${{ needs.release-please.outputs.tag_name }}
+          persist-credentials: false
+
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@4360b52568e2003a75bf9bc1d59f33a8e3fc893c # stable
+
+      - name: Publish omen-cli
+        env:
+          CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2015:info:1:36: Note that A && B || C is not if-then-else. C may run when A is true

(shellcheck)